build/gen_stub.php

```diff
@@ -1117,7 +1117,8 @@ private function initializeValue(string $type, $value, bool $isTyped): string
                 if (empty($value)) {
                     $code .= "\tZVAL_EMPTY_STRING(&$zvalName);\n";
                 } else {
-                    $code .= "\tZVAL_STRING(&$zvalName, \"$value\");\n";
+                    $code .= "\tzend_string *{$zvalName}_str = zend_string_init(\"$value\", sizeof(\"$value\") - 1, 1);\n";
```

Comment:
I'm unsure if I should use this code or the code found in `zend_declare_property_string()`:
```
ZVAL_NEW_STR(&property, zend_string_init(value, strlen(value), ce->type & ZEND_INTERNAL_CLASS));
```